charts/fleet/Chart.yaml

```diff
@@ -8,4 +8,4 @@ version: v5.0.0
 home: https://github.com/fleetdm/fleet
 sources:
   - https://github.com/fleetdm/fleet.git
-appVersion: v5.0.0
```

Comment:
This was [accidentally set](https://github.com/fleetdm/fleet/pull/8420/files#diff-b6587291190d2531e50e09a004c93174b99b4928a2fea4b4b2befc7465ed0d2fL7) to `v5.0.0` previously.